docs/wiki/99 - Developers/SQL Dictionary.md

```diff
@@ -23,3 +23,38 @@
 	| `MRI_alias`  | MRI specific shortname. Limited to 4 characters  |                                                                                              |
 	| `Account`    |                                                  | Deprecated                                                                                              |
 	| `Study_site` | Does this site recruit candidates?             | This field will affect the appearance of the center in dropdowns in several modules of LORIS |
+
+
+- Table: `Project`
+ 
+ 	*This table stores the list of projects configured for the study.*
+ 
+	|         Field        |                  Description                  |                                        Notes                                                 |
+	|:--------------------:|:---------------------------------------------:|:--------------------------------------------------------------------------------------------:|
+	| `ProjectID`          | Identifier of the project                     | Avoid setting this field explicitly when inserting data, it auto increments.                 |
+	| `Name`               | Full name of the project                      |                                                                                              |
+	| `recruitmentTarget`  | Expected number of candidates to be recruited |                                                                                              |
```

Comment:
Maybe add a note in the table to specify that this number is used by the dashboard to compute the recruitment status?

Reply:
euhmm... i dunno. I debated that idea but then we would have to explain each of the other fields where they are used. plus makes it alot harder to maintain in case people start using it for other reasons

I think thtas something you should lookup in the code